src/lava/lib/optimization/utils/solver_benchmarker.py

```diff
@@ -0,0 +1,97 @@
+# INTEL CORPORATION CONFIDENTIAL AND PROPRIETARY
+#
+# Copyright © 2022 Intel Corporation.
+#
+# This software and the related documents are Intel copyrighted
+# materials, and your use of them is governed by the express
+# license under which they were provided to you (License). Unless
+# the License provides otherwise, you may not use, modify, copy,
+# publish, distribute, disclose or transmit  this software or the
+# related documents without Intel's prior written permission.
+#
+# This software and the related documents are provided as is, with
+# no express or implied warranties, other than those that are
+# expressly stated in the License.
+# See: https://spdx.org/licenses/
+# SPDX-License-Identifier: BSD-3-Clause
+try:
+    from lava.utils.loihi2_profiler import Loihi2Power, Loihi2ExecutionTime
+except ImportError:
+    class Loihi2Power:
+        pass
+
+    class Loihi2ExecutionTime:
+        pass
+
+import numpy as np
+
+
+class SolverBenchmarker:
+    """
+    Utility class to benchmark power consumption and execution time of a
+    solver on Loihi2.
+    """
+
+    def __init__(self):
+        self._power_logger = None
+        self._time_logger = None
+
+    def get_power_measurement_cfg(self, num_steps: int):
+        """
+        Returns the pre- and post- fixtures to be passed to a ``Loihi2HwCfg``
+        to enable power consumption monitoring.
+
+        Parameters
```

Comment:
I would suggest to switch to the parameter description that we use as default, in the form of 
![image](https://user-images.githubusercontent.com/86950058/198130018-0445527f-133a-475d-a8e6-2598b328b262.png)

Reply:
Also in the other methods.